OAK-Marketplace-Emotion-Recognition/main.py

```diff
@@ -0,0 +1,24 @@
+import os.path as osp
+from argsparser import parse_args
+from utils.inference import build_model, process_cam
+from utils.helpers import save_video
+from utils.visualization import EmotionBarAnalyzer
+import cv2
+
+
+def main():
+    args = parse_args()
+    model_path = osp.join(args.model_path, "checkpoint")
+    model = build_model(model_path)
```

Comment:
I think we don't need to have the build_model function. It would be much better if we could do something like this for the snippets:

```python
model_path = osp.join(args.model_path, "checkpoint")  # Path to emotion recognition checkpoint

# build model
model = InferenceModel(model_path=model_path)  # Initialize the emotion recognition model
model.model_load()  # Load weights
```